bootstrap.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+sudo yum -y groupinstall 'Development Tools'
+sudo yum -y install tcl 
```

Comment:
@olivermel Nevermind, I'm just running into weird compilation problems in the Prod environment.  It builds fine without this dependency in the VM.